### PR TITLE
[clean] Better (and faster) shutdown process

### PR DESCRIPTION
**Pull request by lpascal-ledger**

### CODE REVIEW

**Review by cedelavergne-ledger — DISMISSED** [review]

File: speculos/api/api.py [hunk 1]
```diff
@@ -23,13 +23,13 @@
 class ApiRunner(IODevice):
     """Run the Speculos API server in a dedicated thread, with a notification when it stops"""
     def __init__(self, api_port: int) -> None:
-        self._app: Flask
+        self._app_wrapper: ApiWrapper
```
Comment (xchapron-ledger) [inline speculos/api/api.py:26]: Unused? And mismatch `_api_wrapper` below?

Comment (lpascal-ledger) [reply speculos/api/api.py:26]: Mismatch true

File: speculos/mcu/struct.py [hunk 2]
```diff
@@ -1,20 +1,15 @@
...
 
 @dataclass
 class Model:
     name: str
-    screen_size: Tuple[int, int]
-    box_position: Tuple[int, int]
-    box_size: Tuple[int, int]
+    screen_size: Pixel
+    box_position: Pixel
+    box_size: Pixel
```
Comment (xchapron-ledger) [inline speculos/mcu/struct.py:12]: screen_size and box_size are more about number of pixels (width, height) than a Pixel which is a x and y coordinate. But that works the same.

**Review by xchapron-ledger — APPROVED** [review]

**Review by xchapron-ledger — APPROVED** [review]

File: speculos/mcu/struct.py (hunk 2 above)
Comment (lpascal-ledger) [reply speculos/mcu/struct.py]: Indeed. I'm changing it as it may be the same type, but is semantically different to a Pixel.

**Pull request merged**